src/brs/http/GetConstants.java

```diff
@@ -14,169 +15,67 @@
-    subtypes.add(subtype);
-    subtype = new JsonObject();
-    subtype.addProperty("value", TransactionType.DigitalGoods.PURCHASE.getSubtype());
-    subtype.addProperty("description", "Purchase");
-    subtypes.add(subtype);
-    subtype = new JsonObject();
-    subtype.addProperty("value", TransactionType.DigitalGoods.DELIVERY.getSubtype());
-    subtype.addProperty("description", "Delivery");
-    subtypes.add(subtype);
-    subtype = new JsonObject();
-    subtype.addProperty("value", TransactionType.DigitalGoods.FEEDBACK.getSubtype());
-    subtype.addProperty("description", "Feedback");
-    subtypes.add(subtype);
-    subtype = new JsonObject();
-    subtype.addProperty("value", TransactionType.DigitalGoods.REFUND.getSubtype());
-    subtype.addProperty("description", "Refund");
-    subtypes.add(subtype);
-    transactionType.add("subtypes", subtypes);
-    transactionTypes.add(transactionType);
-    transactionType = new JsonObject();
-    transactionType.addProperty("value", TransactionType.AccountControl.EFFECTIVE_BALANCE_LEASING.getType());
-    transactionType.addProperty("description", "Account Control");
-    subtypes = new JsonArray();
-    subtype = new JsonObject();
-    subtype.addProperty("value", TransactionType.AccountControl.EFFECTIVE_BALANCE_LEASING.getSubtype());
-    subtype.addProperty("description", "Effective balance leasing");
-    subtypes.add(subtype);
-    transactionType.add("subtypes", subtypes);
-    transactionTypes.add(transactionType);
-    response.add("transactionTypes", transactionTypes);
+        JsonArray peerStates = new JsonArray();
+        JsonObject peerState = new JsonObject();
+        peerState.addProperty("value", 0);
+        peerState.addProperty("description", "Non-connected");
+        peerStates.add(peerState);
+        peerState = new JsonObject();
+        peerState.addProperty("value", 1);
+        peerState.addProperty("description", "Connected");
+        peerStates.add(peerState);
+        peerState = new JsonObject();
+        peerState.addProperty("value", 2);
+        peerState.addProperty("description", "Disconnected");
+        peerStates.add(peerState);
+        response.add("peerStates", peerStates);
 
-    JsonArray peerStates = new JsonArray();
-    JsonObject peerState = new JsonObject();
-    peerState.addProperty("value", 0);
-    peerState.addProperty("description", "Non-connected");
-    peerStates.add(peerState);
-    peerState = new JsonObject();
-    peerState.addProperty("value", 1);
-    peerState.addProperty("description", "Connected");
-    peerStates.add(peerState);
-    peerState = new JsonObject();
-    peerState.addProperty("value", 2);
-    peerState.addProperty("description", "Disconnected");
-    peerStates.add(peerState);
-    response.add("peerStates", peerStates);
+        JsonObject requestTypes = new JsonObject();
+        // for (Map.Entry<String, APIServlet.APIRequestHandler> handlerEntry : APIServlet.apiRequestHandlers.entrySet()) {
```

Comment:
commented code?

Reply:
Yeah needs to be removed but outside of scope of this PR, there are many many instances of commented code like this